api/envoy/type/percent.proto

```diff
@@ -46,3 +46,18 @@ message FractionalPercent {
   // fractional percentage is capped at 1 (100%).
   DenominatorType denominator = 2 [(validate.rules).enum.defined_only = true];
 }
+
+// Runtime derived FractionalPercent with defaults for when the numerator or denominator is not
+// specified via a runtime key.
+message RuntimeFractionalPercent {
```

Comment:
can we move this back into route.proto or create a runtime proto? i don't think percent.proto should contain this runtime concept.